src/index/index.h

```diff
@@ -80,6 +80,9 @@ struct nxs_index {
 	nxs_t *			nxs;
 	char *			name;
 	TAILQ_ENTRY(nxs_index)	entry;
+
+	/* Index params. */
+	nxs_params_t *params;
```

Comment:
Please move this just after the `nxs` and use a tab after `*` align it neatly :)